deploy/cr.yaml

```diff
@@ -556,6 +556,24 @@ spec:
       requests:
         memory: 100M
         cpu: 200m
+
+#  users:
+#  - name: my-user
+#    dbs:
+#    - db1
+#    - db2
+#    hosts:
+#    - localhost
+#    grants:
+#    - SELECT
+#    - DELETE
+#    - INSERT
+#    withGrantOption: true
+#    passwordSecretRef:
+#      name: my-user-pwd
+#      key: my-user-pwd-key
+#  - name: my-user-two
```

Comment:
is it possible to specify only user name and operator will take care of the rest?

Reply:
Yes, in this case the DB will be `*` and grants will be the default ones you get `usage on *.*`. 
And the secret will be created with a generated user password.